mslib/mswms/mswms.py

```diff
@@ -46,6 +46,23 @@ def main():
     parser.add_argument("--logfile", help="If set to a name log output goes to that file", dest="logfile",
                         default=None)
     parser.add_argument("--update", help="Updates MSS to the newest version", action="store_true", default=False)
+
+    subparsers = parser.add_subparsers(help='Available actions', dest='action')
```

Comment:
I am not sure I like subparsers. At least not with many "commands". Examples are "git".

Reply:
I disagree. I find subparsers for commands which inherently belong together reasonable.

Reply:
Fine for me.

Reply:
And my comment was misleading, rereading it. For `git` I find it totally fine and the only way to handle it, but for a *single* action, it might be a bit confusing.